### ISSUE TITLE
ERR_CONNECTION_REFUSED on custom domains

### ISSUE THREAD

**Issue by worldwidejamie** [lando/lando#3503]
I made some updates to my mac (M1), and custom domains have stopped working. The things I've changed:
- Upgrade to Ventura
- Upgrade to Lando 3.10
- Upgrade Docker to 4.16.2

localhost works.

- I've tried updating my hosts file so that `127.0.0.1` points to the custom domain, but that didn't help.
-  I've tried rolling back Lando to 3.8, but that did not resolve the issue either.

 Hopefully, the following information is enough for some context on my setup.

## Landofile
```
name: lando-ac
recipe: wordpress
config:
  webroot: .
  php: 7.4
  composer_version: 2
  database: mariadb
services:
  myadmin:
    type: phpmyadmin
```
----
## Lando Config

```
{ alliance: false,
  appEnv:
   { COLUMNS: 256,
     LANDO: 'ON',
     LANDO_CA_CERT: '/lando/certs/lndo.site.pem',
     LANDO_CA_KEY: '/lando/certs/lndo.site.key',
     LANDO_CONFIG_DIR: '/Users/default-admin/.lando',
     LANDO_DOMAIN: 'lndo.site',
     LANDO_HOST_GID: '20',
     LANDO_HOST_HOME: '/Users/default-admin',
     LANDO_HOST_IP: 'host.docker.internal',
     LANDO_HOST_OS: 'darwin',
     LANDO_HOST_UID: '501',
     LANDO_HOST_USER: 'default-admin',
     LANDO_LEIA: 0,
     LANDO_MOUNT: '/app',
     LANDO_WEBROOT_GROUP: 'www-data',
     LANDO_WEBROOT_USER: 'www-data',
     TERM: 'xterm' },
  appLabels:
   { 'io.lando.container': 'TRUE',
     'io.lando.id': '1607ea138eff46adb89cbf648a0fe8590d2cb4ac' },
  bindAddress: '127.0.0.1',
  caCert: '/Users/default-admin/.lando/certs/lndo.site.pem',
  caDomain: 'lndo.site',
  caKey: '/Users/default-admin/.lando/certs/lndo.site.key',
  caProject: 'landocasetupkenobi38ahsoka1607ea138eff46adb89cbf648a0fe8590d2cb4ac',
  channel: 'stable',
  command: { '$0': '/usr/local/bin/lando', _: [ 'config' ], path: null, v: 0, verbose: 0 },
  composeBin:
   '/Applications/Docker.app/Contents/Resources/bin/docker-compose-v1/docker-compose',
  configSources:
   [ '/snapshot/cli/config.yml',
     '/Users/default-admin/.lando/config.yml',
     { data: {},
       file: '/Users/default-admin/lando-projects/marketing-site/.lando.yml',
       landoFile: true } ],
  disablePlugins: [ 'lando-alliance' ],
  dockerBin: '/Applications/Docker.app/Contents/Resources/bin/docker',
  dockerBinDir: '/Applications/Docker.app/Contents/Resources/bin',
  dockerSupportedVersions:
   { compose:
      { link:
         { linux: 'https://docs.docker.com/compose/install/#install-compose-on-linux-systems' },
        max: '1.30.0',
        min: '1.23.0' },
     desktop:
      { link:
         { darwin: 'https://docs.docker.com/docker-for-mac/release-notes/',
           win32: 'https://docs.docker.com/docker-for-windows/release-notes/' },
        max: '4.16.99',
        min: '2.1.0.0' },
     engine:
      { link: { linux: 'https://docs.docker.com/engine/install/' },
        max: '20.10.99',
        min: '18.09.3' } },
  domain: 'lndo.site',
  engineConfig: { host: '127.0.0.1', port: 2376, socketPath: '/var/run/docker.sock' },
  env:
   { AC_PLATFORM_NETWORK_NAME: 'k3d-ac-platform',
     AC_PLATFORM_RESOLV_CONFIG: '/Users/default-admin/.activecampaign/ac-platform/docker-network-resolv.conf',
     COLORTERM: 'truecolor',
     COMMAND_MODE: 'unix2003',
     COMPOSE_HTTP_TIMEOUT: '300',
     CONDA_CHANGEPS1: 'false',
     HOME: '/Users/default-admin',
     HOMEBREW_CELLAR: '/opt/homebrew/Cellar',
     HOMEBREW_PREFIX: '/opt/homebrew',
     HOMEBREW_REPOSITORY: '/opt/homebrew',
     INFOPATH: '/opt/homebrew/share/info:',
     JAVA_HOME: '/Users/default-admin/.activecampaign/java/current',
     LC_ALL: 'en_US.UTF-8',
     LESS: '-R',
     LOGNAME: 'default-admin',
     LSCOLORS: 'Gxfxcxdxbxegedabagacad',
     LaunchInstanceID: '033652A3-AAFB-4262-A835-2EF890376510',
     MANPATH:
      '/Users/default-admin/.nvm/versions/node/v16.10.0/share/man:/opt/homebrew/share/man::',
     NVM_BIN: '/Users/default-admin/.nvm/versions/node/v16.10.0/bin',
     NVM_CD_FLAGS: '-q',
     NVM_DIR: '/Users/default-admin/.nvm',
     NVM_INC: '/Users/default-admin/.nvm/versions/node/v16.10.0/include/node',
     PAGER: 'less',
     PATH:
      '/Users/default-admin/.activecampaign/java/current/bin:/Users/default-admin/.activecampaign/bin:/Users/default-admin/.activecampaign/kafka/bin:/usr/local/opt/php/bin:/usr/local/opt/php@7.4/bin:/Users/default-admin/.nvm/versions/node/v16.10.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin',
     PWD:
      '/Users/default-admin/lando-projects/marketing-site/wp-content/themes/activecampaign',
     SECURITYSESSIONID: '186b2',
     SHELL: '/bin/zsh',
     SHLVL: '1',
     SSH_AUTH_SOCK: '/private/tmp/com.apple.launchd.ynEkXV8vMZ/Listeners',
     TERM: 'xterm-256color',
     TERM_PROGRAM: 'WarpTerminal',
     TERM_PROGRAM_VERSION: 'v0.2023.01.17.08.03.stable_01',
     TMPDIR: '/var/folders/jh/40wlx29j31bg3cly17n7b_z40000gn/T/',
     USER: 'default-admin',
     WARP_IS_LOCAL_SHELL_SESSION: '1',
     WARP_PS1:
      '%(?:%{\x1B[01;32m%}➜ :%{\x1B[01;31m%}➜ ) %{$fg[cyan]%}%c%{$reset_color%} $(git_prompt_info)',
     WARP_USE_SSH_WRAPPER: '1',
     XPC_FLAGS: '0x0',
     XPC_SERVICE_NAME: '0',
     ZSH: '/Users/default-admin/.oh-my-zsh',
     _: '/usr/local/bin/lando',
     __CFBundleIdentifier: 'dev.warp.Warp-Stable',
     __CF_USER_TEXT_ENCODING: '0x1F5:0x0:0x0' },
  envPrefix: 'LANDO',
  experimental: false,
  gid: '20',
  hconf: '/Users/default-admin/Library/Caches/hyperdrive/lando.json',
  home: '/Users/default-admin',
  hyperdrive: 'hyperdrive',
  id: '91ea4b01-c791-464a-a0da-1d671b1ad712',
  inits:
   [ { build: [Function: build],
       name: 'acquia',
       options: [Function: options],
       overrides: { name: { when: [Function: when] }, webroot: { when: [Function: when] } },
       sources:
        [ { build: [Function: build],
            label: 'acquia',
            name: 'acquia',
            overrides: { recipe: { when: [Function: when] } } } ] },
     { name: 'backdrop' },
     { name: 'drupal10' },
     { name: 'drupal6' },
     { name: 'drupal7' },
     { name: 'drupal8' },
     { name: 'drupal9' },
     { name: 'joomla' },
     { build: [Function: build],
       name: 'lagoon',
       options: [Function: options],
       overrides: { name: { when: [Function: when] }, webroot: { when: [Function: when] } },
       sources:
        [ { build: [Function: build],
            label: 'lagoon',
            name: 'lagoon',
            overrides: { recipe: { when: [Function: when] } } } ] },
     { name: 'lamp' },
     { name: 'laravel' },
     { name: 'lemp' },
     { name: 'mean', overrides: { webroot: { when: [Function: when] } } },
     { build: [Function: build],
       name: 'pantheon',
       options: [Function: options],
       overrides: { name: { when: [Function: when] }, webroot: { when: [Function: when] } },
       sources:
        [ { build: [Function: build],
            label: 'pantheon',
            name: 'pantheon',
            overrides: { recipe: { when: [Function: when] } } } ] },
     { build: [Function: build],
       name: 'platformsh',
       options: [Function: options],
       overrides: { name: { when: [Function: when] }, webroot: { when: [Function: when] } },
       sources:
        [ { build: [Function: build],
            label: 'platformsh',
            name: 'platformsh',
            overrides: { recipe: { when: [Function: when] } } } ] },
     { name: 'symfony' },
     { name: 'wordpress' } ],
  instance: '1607ea138eff46adb89cbf648a0fe8590d2cb4ac',
  isArmed: true,
  landoFile: '.lando.yml',
  landoFileConfig:
   { composeCache: '/Users/default-admin/.lando/cache/lando-ac.compose.cache',
     config: { composer_version: 2, database: 'mariadb', php: 7.4, webroot: '.' },
     configFiles: [ '/Users/default-admin/lando-projects/marketing-site/.lando.yml' ],
     metaCache: 'lando-ac.meta.cache',
     name: 'lando-ac',
     project: 'landoac',
     recipe: 'wordpress',
     root: '/Users/default-admin/lando-projects/marketing-site',
     services: { myadmin: { type: 'phpmyadmin' } },
     tooling:
      { 'db-export [file]':
         { cmd: '/helpers/sql-export.sh',
           description: 'Exports database from a database service to a file',
           options:
            { host:
               { alias: [ 'h' ], default: 'database', description: 'The database service to use' },
              stdout: { description: 'Dump database to stdout' } },
           service: ':host',
           user: 'root' },
        'db-import <file>':
         { cmd: '/helpers/sql-import.sh',
           description: 'Imports a dump file into a database service',
           options:
            { 'no-wipe':
               { boolean: true,
                 description: 'Do not destroy the existing database before an import' },
              host:
               { alias: [ 'h' ], default: 'database', description: 'The database service to use' } },
           service: ':host',
           user: 'root' },
        composer: { cmd: 'composer --ansi', service: 'appserver' },
        mysql:
         { cmd: 'mysql -uroot',
           description: 'Drops into a MySQL shell on a database service',
           options:
            { host:
               { alias: [ 'h' ], default: 'database', description: 'The database service to use' } },
           service: ':host' },
        php: { cmd: 'php', service: 'appserver' },
        wp: { service: 'appserver' } },
     toolingCache: '/Users/default-admin/.lando/cache/lando-ac.tooling.cache',
     toolingRouter: '/Users/default-admin/.lando/cache/lando-ac.tooling.router' },
  leia: false,
  logDir: '/Users/default-admin/.lando/logs',
  logLevel: 'debug',
  logLevelConsole: 'warn',
  maxKeyWarning: 10,
  mode: 'cli',
  networkBridge: 'lando_bridge_network',
  node: 'v16.14.2',
  os: { arch: 'arm64', platform: 'darwin', release: '22.1.0', type: 'Darwin' },
  packaged: true,
  pluginDirs:
   [ '/snapshot/cli',
     '/snapshot/cli/node_modules/@lando/core',
     { dir: '/snapshot/cli/integrations',
       path: '/snapshot/cli/integrations',
       subdir: '.' },
     { dir: '/snapshot/cli/node_modules/@lando',
       namespace: '@lando',
       path: '/snapshot/cli/node_modules/@lando',
       subdir: '.' },
     { dir: '/Users/default-admin/.lando/global-plugins/@lando',
       namespace: '@lando',
       path: '/Users/default-admin/.lando/global-plugins/@lando',
       subdir: '.' },
     { dir: '/Users/default-admin/.lando/plugins/@lando',
       namespace: '@lando',
       path: '/Users/default-admin/.lando/plugins/@lando',
       subdir: '.' },
     '/Users/default-admin/.lando' ],
  plugins:
   [ { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/app.js',
       compose: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/compose',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core',
       name: 'lando-core',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/index.js',
       scripts: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core',
       services: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/services',
       tasks: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/tasks',
       types: '/snapshot/cli/node_modules/@lando/core/plugins/lando-core/types' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-events/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-events',
       name: 'lando-events',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-events/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-events' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-networking/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-networking',
       name: 'lando-networking',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-networking/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-networking' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy',
       name: 'lando-proxy',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy/index.js',
       scripts: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy',
       services: '/snapshot/cli/node_modules/@lando/core/plugins/lando-proxy/services' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes',
       name: 'lando-recipes',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/index.js',
       scripts: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes',
       services: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/services',
       sources: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/sources',
       tasks: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/tasks',
       types: '/snapshot/cli/node_modules/@lando/core/plugins/lando-recipes/types' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-services/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-services',
       name: 'lando-services',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-services/index.js',
       scripts: '/snapshot/cli/node_modules/@lando/core/plugins/lando-services/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-services' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing',
       name: 'lando-sharing',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing/index.js',
       scripts: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing',
       services: '/snapshot/cli/node_modules/@lando/core/plugins/lando-sharing/services' },
     { dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-test',
       name: 'lando-test',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-test/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-test' },
     { app: '/snapshot/cli/node_modules/@lando/core/plugins/lando-tooling/app.js',
       dir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-tooling',
       name: 'lando-tooling',
       path: '/snapshot/cli/node_modules/@lando/core/plugins/lando-tooling/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/core/plugins/lando-tooling',
       tasks: '/snapshot/cli/node_modules/@lando/core/plugins/lando-tooling/tasks' },
     { app: '/snapshot/cli/node_modules/@lando/acquia/app.js',
       dir: '/snapshot/cli/node_modules/@lando/acquia',
       name: '@lando/acquia',
       path: '/snapshot/cli/node_modules/@lando/acquia/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/acquia/recipes',
       scripts: '/snapshot/cli/node_modules/@lando/acquia/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/acquia' },
     { dir: '/snapshot/cli/node_modules/@lando/apache',
       name: '@lando/apache',
       path: '/snapshot/cli/node_modules/@lando/apache/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/apache',
       services: '/snapshot/cli/node_modules/@lando/apache/services' },
     { dir: '/snapshot/cli/node_modules/@lando/backdrop',
       name: '@lando/backdrop',
       path: '/snapshot/cli/node_modules/@lando/backdrop/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/backdrop/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/backdrop' },
     { dir: '/snapshot/cli/node_modules/@lando/compose',
       name: '@lando/compose',
       path: '/snapshot/cli/node_modules/@lando/compose/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/compose',
       services: '/snapshot/cli/node_modules/@lando/compose/services' },
     { dir: '/snapshot/cli/node_modules/@lando/dotnet',
       name: '@lando/dotnet',
       path: '/snapshot/cli/node_modules/@lando/dotnet/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/dotnet',
       services: '/snapshot/cli/node_modules/@lando/dotnet/services' },
     { dir: '/snapshot/cli/node_modules/@lando/drupal',
       name: '@lando/drupal',
       path: '/snapshot/cli/node_modules/@lando/drupal/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/drupal/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/drupal' },
     { dir: '/snapshot/cli/node_modules/@lando/elasticsearch',
       name: '@lando/elasticsearch',
       path: '/snapshot/cli/node_modules/@lando/elasticsearch/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/elasticsearch',
       services: '/snapshot/cli/node_modules/@lando/elasticsearch/services' },
     { dir: '/snapshot/cli/node_modules/@lando/go',
       name: '@lando/go',
       path: '/snapshot/cli/node_modules/@lando/go/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/go',
       services: '/snapshot/cli/node_modules/@lando/go/services' },
     { dir: '/snapshot/cli/node_modules/@lando/joomla',
       name: '@lando/joomla',
       path: '/snapshot/cli/node_modules/@lando/joomla/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/joomla/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/joomla' },
     { app: '/snapshot/cli/node_modules/@lando/lagoon/app.js',
       dir: '/snapshot/cli/node_modules/@lando/lagoon',
       name: '@lando/lagoon',
       path: '/snapshot/cli/node_modules/@lando/lagoon/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/lagoon/recipes',
       scripts: '/snapshot/cli/node_modules/@lando/lagoon/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/lagoon',
       services: '/snapshot/cli/node_modules/@lando/lagoon/services',
       types: '/snapshot/cli/node_modules/@lando/lagoon/types' },
     { dir: '/snapshot/cli/node_modules/@lando/lamp',
       name: '@lando/lamp',
       path: '/snapshot/cli/node_modules/@lando/lamp/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/lamp/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/lamp' },
     { dir: '/snapshot/cli/node_modules/@lando/laravel',
       name: '@lando/laravel',
       path: '/snapshot/cli/node_modules/@lando/laravel/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/laravel/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/laravel' },
     { dir: '/snapshot/cli/node_modules/@lando/lemp',
       name: '@lando/lemp',
       path: '/snapshot/cli/node_modules/@lando/lemp/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/lemp/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/lemp' },
     { dir: '/snapshot/cli/node_modules/@lando/mailhog',
       name: '@lando/mailhog',
       path: '/snapshot/cli/node_modules/@lando/mailhog/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/mailhog',
       services: '/snapshot/cli/node_modules/@lando/mailhog/services' },
     { dir: '/snapshot/cli/node_modules/@lando/mariadb',
       name: '@lando/mariadb',
       path: '/snapshot/cli/node_modules/@lando/mariadb/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/mariadb',
       services: '/snapshot/cli/node_modules/@lando/mariadb/services' },
     { dir: '/snapshot/cli/node_modules/@lando/mean',
       name: '@lando/mean',
       path: '/snapshot/cli/node_modules/@lando/mean/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/mean/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/mean' },
     { dir: '/snapshot/cli/node_modules/@lando/memcached',
       name: '@lando/memcached',
       path: '/snapshot/cli/node_modules/@lando/memcached/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/memcached',
       services: '/snapshot/cli/node_modules/@lando/memcached/services' },
     { dir: '/snapshot/cli/node_modules/@lando/mongo',
       name: '@lando/mongo',
       path: '/snapshot/cli/node_modules/@lando/mongo/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/mongo',
       services: '/snapshot/cli/node_modules/@lando/mongo/services' },
     { dir: '/snapshot/cli/node_modules/@lando/mssql',
       name: '@lando/mssql',
       path: '/snapshot/cli/node_modules/@lando/mssql/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/mssql',
       services: '/snapshot/cli/node_modules/@lando/mssql/services' },
     { dir: '/snapshot/cli/node_modules/@lando/mysql',
       name: '@lando/mysql',
       path: '/snapshot/cli/node_modules/@lando/mysql/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/mysql',
       services: '/snapshot/cli/node_modules/@lando/mysql/services' },
     { dir: '/snapshot/cli/node_modules/@lando/nginx',
       name: '@lando/nginx',
       path: '/snapshot/cli/node_modules/@lando/nginx/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/nginx',
       services: '/snapshot/cli/node_modules/@lando/nginx/services' },
     { dir: '/snapshot/cli/node_modules/@lando/node',
       name: '@lando/node',
       path: '/snapshot/cli/node_modules/@lando/node/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/node',
       services: '/snapshot/cli/node_modules/@lando/node/services' },
     { app: '/snapshot/cli/node_modules/@lando/pantheon/app.js',
       dir: '/snapshot/cli/node_modules/@lando/pantheon',
       name: '@lando/pantheon',
       path: '/snapshot/cli/node_modules/@lando/pantheon/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/pantheon/recipes',
       scripts: '/snapshot/cli/node_modules/@lando/pantheon/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/pantheon',
       services: '/snapshot/cli/node_modules/@lando/pantheon/services' },
     { dir: '/snapshot/cli/node_modules/@lando/php',
       name: '@lando/php',
       path: '/snapshot/cli/node_modules/@lando/php/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/php',
       services: '/snapshot/cli/node_modules/@lando/php/services' },
     { dir: '/snapshot/cli/node_modules/@lando/phpmyadmin',
       name: '@lando/phpmyadmin',
       path: '/snapshot/cli/node_modules/@lando/phpmyadmin/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/phpmyadmin',
       services: '/snapshot/cli/node_modules/@lando/phpmyadmin/services' },
     { app: '/snapshot/cli/node_modules/@lando/platformsh/app.js',
       dir: '/snapshot/cli/node_modules/@lando/platformsh',
       name: '@lando/platformsh',
       path: '/snapshot/cli/node_modules/@lando/platformsh/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/platformsh/recipes',
       scripts: '/snapshot/cli/node_modules/@lando/platformsh/scripts',
       searchDir: '/snapshot/cli/node_modules/@lando/platformsh',
       services: '/snapshot/cli/node_modules/@lando/platformsh/services',
       types: '/snapshot/cli/node_modules/@lando/platformsh/types' },
     { dir: '/snapshot/cli/node_modules/@lando/postgres',
       name: '@lando/postgres',
       path: '/snapshot/cli/node_modules/@lando/postgres/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/postgres',
       services: '/snapshot/cli/node_modules/@lando/postgres/services' },
     { dir: '/snapshot/cli/node_modules/@lando/python',
       name: '@lando/python',
       path: '/snapshot/cli/node_modules/@lando/python/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/python',
       services: '/snapshot/cli/node_modules/@lando/python/services' },
     { dir: '/snapshot/cli/node_modules/@lando/redis',
       name: '@lando/redis',
       path: '/snapshot/cli/node_modules/@lando/redis/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/redis',
       services: '/snapshot/cli/node_modules/@lando/redis/services' },
     { dir: '/snapshot/cli/node_modules/@lando/ruby',
       name: '@lando/ruby',
       path: '/snapshot/cli/node_modules/@lando/ruby/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/ruby',
       services: '/snapshot/cli/node_modules/@lando/ruby/services' },
     { dir: '/snapshot/cli/node_modules/@lando/solr',
       name: '@lando/solr',
       path: '/snapshot/cli/node_modules/@lando/solr/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/solr',
       services: '/snapshot/cli/node_modules/@lando/solr/services' },
     { dir: '/snapshot/cli/node_modules/@lando/symfony',
       name: '@lando/symfony',
       path: '/snapshot/cli/node_modules/@lando/symfony/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/symfony/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/symfony' },
     { dir: '/snapshot/cli/node_modules/@lando/tomcat',
       name: '@lando/tomcat',
       path: '/snapshot/cli/node_modules/@lando/tomcat/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/tomcat',
       services: '/snapshot/cli/node_modules/@lando/tomcat/services' },
     { dir: '/snapshot/cli/node_modules/@lando/varnish',
       name: '@lando/varnish',
       path: '/snapshot/cli/node_modules/@lando/varnish/index.js',
       searchDir: '/snapshot/cli/node_modules/@lando/varnish',
       services: '/snapshot/cli/node_modules/@lando/varnish/services' },
     { dir: '/snapshot/cli/node_modules/@lando/wordpress',
       name: '@lando/wordpress',
       path: '/snapshot/cli/node_modules/@lando/wordpress/index.js',
       recipes: '/snapshot/cli/node_modules/@lando/wordpress/recipes',
       searchDir: '/snapshot/cli/node_modules/@lando/wordpress' } ],
  postLandoFiles: [ '.lando.local.yml', '.lando.user.yml' ],
  preLandoFiles:
   [ '.lando.base.yml', '.lando.dist.yml', '.lando.recipe.yml', '.lando.upstream.yml' ],
  process: 'node',
  product: 'lando',
  proxy: 'ON',
  proxyBindAddress: '127.0.0.1',
  proxyCache: 'proxyCache',
  proxyCommand:
   [ '/entrypoint.sh',
     '--log.level=DEBUG',
     '--api.insecure=true',
     '--api.dashboard=false',
     '--providers.docker=true',
     '--entrypoints.https.address=:443',
     '--entrypoints.http.address=:80',
     '--providers.docker.exposedbydefault=false',
     '--providers.file.directory=/proxy_config',
     '--providers.file.watch=true' ],
  proxyConfigDir: '/Users/default-admin/.lando/proxy/config',
  proxyContainer: 'landoproxyhyperion5000gandalfedition_proxy_1',
  proxyCurrentPorts: { http: '80', https: '443' },
  proxyCustom: {},
  proxyDefaultCert: '/certs/cert.crt',
  proxyDefaultKey: '/certs/cert.key',
  proxyDir: '/Users/default-admin/.lando/proxy',
  proxyDomain: 'lndo.site',
  proxyHttpFallbacks: [ '8000', '8080', '8888', '8008' ],
  proxyHttpPort: '80',
  proxyHttpPorts: [ '80', '8000', '8080', '8888', '8008' ],
  proxyHttpsFallbacks: [ '444', '4433', '4444', '4443' ],
  proxyHttpsPort: '443',
  proxyHttpsPorts: [ '443', '444', '4433', '4444', '4443' ],
  proxyIp: '127.0.0.1',
  proxyLastPorts: { http: '8000', https: '443' },
  proxyName: 'landoproxyhyperion5000gandalfedition',
  proxyNet: 'landoproxyhyperion5000gandalfedition_edge',
  proxyPassThru: true,
  proxyScanHttp:
   [ 'http://127.0.0.1:80',
     'http://127.0.0.1:8000',
     'http://127.0.0.1:8080',
     'http://127.0.0.1:8888',
     'http://127.0.0.1:8008' ],
  proxyScanHttps:
   [ 'https://127.0.0.1:443',
     'https://127.0.0.1:444',
     'https://127.0.0.1:4433',
     'https://127.0.0.1:4444',
     'https://127.0.0.1:4443' ],
  recipes:
   [ 'acquia',
     'backdrop',
     'drupal10',
     'drupal6',
     'drupal7',
     'drupal8',
     'drupal9',
     'joomla',
     'lagoon',
     'lamp',
     'laravel',
     'lemp',
     'mean',
     'pantheon',
     'platformsh',
     'symfony',
     'wordpress' ],
  runtime: 3,
  sources:
   [ { build: [Function: build],
       label: 'acquia',
       name: 'acquia',
       overrides: { recipe: { when: [Function: when] } } },
     { label: 'current working directory', name: 'cwd' },
     { build: [Function: build],
       label: 'github',
       name: 'github',
       options: [Function: options] },
     { build: [Function: build],
       label: 'lagoon',
       name: 'lagoon',
       overrides: { recipe: { when: [Function: when] } } },
     { build: [Function: build],
       label: 'pantheon',
       name: 'pantheon',
       overrides: { recipe: { when: [Function: when] } } },
     { build: [Function: build],
       label: 'platformsh',
       name: 'platformsh',
       overrides: { recipe: { when: [Function: when] } } },
     { build: [Function: build],
       label: 'remote git repo or archive',
       name: 'remote',
       options: [Function: options] } ],
  stats: [ { report: true, url: 'https://metrics.lando.dev' } ],
  uid: '501',
  user: '91ea4b01-c791-464a-a0da-1d671b1ad712',
  userAgent: 'Lando/3.10.0',
  userConfRoot: '/Users/default-admin/.lando',
  username: 'default-admin',
  version: '3.10.0' }
```

I'm really at a loss for what more I could do to attempt troubleshooting this issue. My current assumption is that this has something to do with Ventura.

**Comment by worldwidejamie** [comment]
This problem was resolved after I updated to Ventura 13.2

**Comment by worldwidejamie** [comment]
This problem was resolved after I updated to Ventura 13.2
